docs/guides/admin/docs/modules/paella.player7/configuration.md

```diff
@@ -0,0 +1,42 @@
+Paella Player 7
+===============
+
+The Paella `(pronounced 'paeja')` [Player](https://paellaplayer.upv.es) is an Open Source
+JavaScript video player capable of playing an unlimited number of audio & video streams 
+synchronously, Live Streaming, Zoom, Captions, contributed user plugins and a lot more. 
+It is easy to install and customize for your own needs.
+
+Paella 7 will be a complete rewrite of Paella, aiming several issues
+
+- Allow easier integration in other platforms by leaving out the singleton design pattern.
+- Easier styling and accessibility support.
+- Reduce the number of dependencies.
+- Allow easier long-term maintenance of the project.
+- Solve the technical debt of 10 years of development.
+
+Have a look at the paella 7 [repository](https://github.com/polimediaupv/paella-core)
+or [documentation page](https://github.com/polimediaupv/paella-core/blob/main/doc/index.md).
+
+Warning
+-------
+
+Paella 7 is still in beta and this module should not be used in production. 
+It is intended to test the new features of paella 7.
```

Comment:
Now that is ready, you can delete that part. :)

Reply:
Done!